content/en/real_user_monitoring/error_tracking/mobile/android.md

```diff
@@ -39,6 +39,31 @@ If you have not set up the Android SDK yet, follow the [in-app setup instruction
 
 For any given error, you can access the file path, line number, and a code snippet for each frame of the related stack trace.
 
+### Report ANRs
+
+An "Application Not Responding" ([ANR][6]) is an Android-specific type of error that gets triggered when the application is unresponsive for too long.
+
+ANRs are only reported through the RUM SDK (not through Logs).
+
+#### Non-fatal ANR reporting
```

Comment:
Shouldn't we write ANR**s** to remain consistent?

Reply:
The singular looks better to me. To maintain consistency, the earlier header can say "Report ANR errors"

Reply:
Whatever you prefer between singular or plural, just need to make it consistent :)

And I'm happy with:
- Report fatal ANR
- Report non-fatal ANR

But I feel that "Report fatal ANR error" is a bit redundant

Reply:
Will update to Report fatal ANR and Report non-fatal ANR. Thank you both for the input!